src/v/cluster/feature_manager.cc

```diff
@@ -226,7 +226,8 @@ ss::future<> feature_manager::maybe_log_license_check_info() {
                      none;
         };
         auto has_roles = !_role_store.local()
-                            .range([](auto const&) { return true; })
+                            .range(
+                              security::role_store::exclude_default_role_filter)
                             .empty();
```

Comment:
suggestion (non-blocking): could use `role_store::size()` and `role_store::contains(default_role)` to avoid the allocating the range query result.